project/dexter-daemon/src/java/com/samsung/sec/dexter/daemon/DexterDaemonActivator.java

```diff
@@ -209,6 +214,15 @@ private void startMonitorForDexterConfigFile() {
 		monitorJob.schedule();
 	}
 	
+	private void startMonitorForPlatzKeywordFile(){
```

Comment:
If developers are not in VD, this Job should not be started.
Please check the PLATZ is working, then start this job if the site exists.
(if you cann't change it right now, you can change it for next version)